### PR TITLE
Created footer component

### PR DESCRIPTION
**Pull request by AlexandruGru**

### CODE REVIEW

File: src/components/footer.css [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    flex-direction: column;
+}
+.list {
+    width: 100%;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 21px;
+    color: #FFFFFF;
+    display: flex;
```
Comment (AlexandruGru) [inline src/components/footer.css]: add justify-content: centre

File: src/components/Footer.js [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+import FbIcon from "./images/pngwing.com (2).png"
+import OkIcon from "./images/pngwing.com (3).png"
+import IgIcon from "./images/pngwing.com (4).png"
+import Arrow from "./images/arrow.png"
+
+const Footer = () => {
+
+    const elements = {
+        "sales": {
+            title: ["Продажа"],
```
Comment (AlexandruGru) [inline src/components/Footer.js]: value of field title can be just string, not array. also u should change logic in reurnable elements

File: src/components/Footer.js [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+                            elements.sales.title.map((item) => {
+                                return (
+                                    <h3>{item}</h3>
+                                )
+                            })
+                        }
+                        {
+                            elements.sales.element.map((item) => {
+                                return (
+                                    <li>{item}</li>
```
Comment (AlexandruGru) [inline src/components/Footer.js]: add key to first returnable element in map

File: src/components/Footer.js [hunk 4]
```diff
@@ -0,0 +1,142 @@
+import React from "react";
```
Comment (AlexandruGru) [inline src/components/Footer.js]: create folder footer and add this files into it

File: src/components/footer.css (hunk 1 above)
Comment (ecaterinaursu) [reply src/components/footer.css]: done

**Pull request merged**